doc/introduction/returns.rst

```diff
@@ -0,0 +1,170 @@
+.. _returns:
+
+QNode returns
+=============
+
+Version 0.30.0 of PennyLane updated the return type of a :class:`~.pennylane.QNode`. This page
+provides additional details about the update and can be used to troubleshoot issues.
+
+.. note::
+
+    If you are looking for a quick fix, jump to the :ref:`Troubleshooting` section!
+
+Summary of the update
+---------------------
+
+.. highlights::
+
+    PennyLane QNodes now return exactly what you tell them to! 🎉

```

Comment:
Looks like this might be having a formatting problem? Or is it supposed to look that way?

Reply:
This way?
![image](https://user-images.githubusercontent.com/49409390/233669341-6dca2455-bb6c-4b97-ad05-9ab6f0a238d0.png)

That was the intention, but if you think it looks like a formatting error then maybe it doesn't look good :laughing:

Reply:
Yeah I agree it does look a little strange. Can we just italicize or bold the text instead?

Reply:
@trbromley this is a case where you probably want to use an admonition, like `.. note::` or `.. important::`.

Reply:
Thanks! Now it looks better:
![image](https://user-images.githubusercontent.com/49409390/233717359-1480152e-b8ca-473c-a970-485118861e21.png)

This was using
```rst
.. rst-class:: admonition tip
```